.github/workflows/sdk-update-alert.yml

```diff
@@ -0,0 +1,32 @@
+name: Check Chartboost Mediation SDK Version
+
+on:
+  schedule:
+    # Runs every day at midnight
+    - cron: '0 0 * * *'
```

Comment:
just confirming, this will run at 5 pm pacific I believe

Reply:
yeah I mean it's midnight somewhere (UTC?). they need to support timezones. bump this: https://github.com/orgs/community/discussions/13454 😛